lib/linguist/languages.yml

```diff
@@ -4575,6 +4575,8 @@ Reason:
   extensions:
   - ".re"
   - ".rei"
+  - ".res"
+  - ".resi"
```

Comment:
There are only 18 such files, far from the requirement of hundreds of repositories. Please remove it.

Reply:
Thanks, I removed the `.resi` extension and its file sample.